01-odd-or-even/01-razvan.js

```diff
@@ -0,0 +1,6 @@
+let input = 19;
+if(input % 2 === 0){
```

Comment:
for simple conditions like this. You can use ternary:

input % 2 === 0 ? alert('Even') : alert('Odd');